lib/gringotts/gateways/monei.ex

```diff
@@ -30,7 +30,6 @@ defmodule Gringotts.Gateways.Monei do
   | ----                | ---    | ----            |
   | `billing_address`   |        | Not implemented |
   | `cart`              |        | Not implemented |
-  | `currency`          |        | **Implemented** |
```

Comment:
@oyeb why did we remove this one? these params show what goes in the payments gateway and  this PR should not have any change in the effect? Am I missing something here?

Reply:
`currency` is no longer an optional arg. It's explicitly available with the `amount`.

Reply:
okay, I think I misunderstood that! 👍